### PR TITLE
WRN-3671: Added ContextualPopupDecorator unit tests.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
There were no tests for ContextualPopupDecorator.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added unit tests for ContextualPopupDecorator.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-3671

### CODE REVIEW

File: ContextualPopupDecorator/tests/ContextualPopupDecorator-specs.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+					Hello
+				</ContextualButton>
+			</Root>
+		);
+		const contextualPopup = screen.getByRole('alert').children.item(1);
+
+		const expectedAttribute = 'direction';
+		const expectedValue = 'right';
+
+		expect(contextualPopup).toHaveAttribute(expectedAttribute, expectedValue);
```
Comment (daniel-stoian-lgp) [inline ContextualPopupDecorator/tests/ContextualPopupDecorator-specs.js:116]: the expectation is not the same as the description of the test. can you check the className ? Or update the description

Comment (alexandrumorariu) [reply ContextualPopupDecorator/tests/ContextualPopupDecorator-specs.js:116]: Description is wrong, will fix.

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**